code/modules/paperwork/desk_bell.dm

```diff
@@ -0,0 +1,108 @@
+// A receptionist's bell
+
+/obj/structure/desk_bell
+	name = "desk bell"
+	desc = "The cornerstone of any customer service job. You feel an unending urge to ring it."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "desk_bell"
+	layer = OBJ_LAYER
+	anchored = FALSE
+	pass_flags = PASSTABLE // Able to place on tables
+	max_integrity = 5000 // To make attacking it not instantly break it
+	/// The amount of times this bell has been rang, used to check the chance it breaks
+	var/times_rang = 0
+	/// Is this bell broken?
+	var/broken_ringer = FALSE
+	/// The cooldown for ringing the bell
+	COOLDOWN_DECLARE(ring_cooldown)
+	/// The length of the cooldown. Setting it to 0 will skip all cooldowns alltogether.
+	var/ring_cooldown_length = 0.3 SECONDS // This is here to protect against tinnitus.
+	/// The sound the bell makes
+	var/ring_sound = 'sound/machines/microwave/microwave-end.ogg'
+
+/obj/structure/desk_bell/Initialize(mapload)
+	. = ..()
+	register_context()
+
+/obj/structure/desk_bell/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	. = ..()
+
+	if(held_item?.tool_behaviour == TOOL_WRENCH)
+		context[SCREENTIP_CONTEXT_RMB] = "Disassemble"
+		return CONTEXTUAL_SCREENTIP_SET
+
+	if(broken_ringer)
+		if(held_item?.tool_behaviour == TOOL_SCREWDRIVER)
+			context[SCREENTIP_CONTEXT_LMB] = "Fix"
+
+	else
+		var/click_context = "Ring"
+		if(prob(1))
+			click_context = "Annoy"
+		context[SCREENTIP_CONTEXT_LMB] = click_context
+	return CONTEXTUAL_SCREENTIP_SET
+
+/obj/structure/desk_bell/attack_hand(mob/living/user, list/modifiers)
+	. = ..()
+	if(!COOLDOWN_FINISHED(src, ring_cooldown) && ring_cooldown_length)
+		return TRUE
+	if(!ring_bell(user))
+		to_chat(user, span_notice("[src] is silent. Some idiot broke it."))
+	if(ring_cooldown_length)
+		COOLDOWN_START(src, ring_cooldown, ring_cooldown_length)
+	return TRUE
+
+/obj/structure/desk_bell/attackby(obj/item/weapon, mob/living/user, params)
+	. = ..()
+	times_rang += weapon.force
+	ring_bell(user)
```

Comment:
so we use weapon.force + 1 uses of the bell? this seems iffy

Reply:
I figured it didn't really make a difference, since the probability is measured in such small percents.
Also, if something has 0 force, it needs to still update the bell number by one.

Reply:
dude what, why is this not `+= 1`

Reply:
I'd imagine if a weapon hit a bell it would be much more likely to get busted than if you rang it normally.